python-libraries/nanover-core/tests/imd/test_imd_client.py

```diff
@@ -14,6 +16,30 @@
 IMMEDIATE_REPLY_WAIT_TIME = 0.01
 
 
+def assert_equal_soon(a: Callable, b: Callable, interval=0.1, timeout=1.0):
+    __tracebackhide__ = True
```

Comment:
This is some pytest magic I did not know about. It may be good to add a comment explaining that it makes pytest hide the function in the traceback, maybe link to the pytest doc, and explaining the rationale of the hiding here.

Does the function appearing in the traceback really make a difference, though?

Reply:
this is the difference:
![image](https://github.com/IRL2/nanover-protocol/assets/527158/ca2b396e-fbd6-4477-8bd9-ab256d5d09d3)
![image](https://github.com/IRL2/nanover-protocol/assets/527158/dd94f196-f5ff-46fb-a7ad-b12d53d2491b)